internal/git/commands_test.go

```diff
@@ -1410,29 +1410,4 @@ func TestBackendCommands(t *testing.T) {
 			must.EqOp(t, want, have)
 		}
 	})
-
-	t.Run("parseActiveBranchDuringRebase", func(t *testing.T) {
-		t.Parallel()
-		t.Run("branch name is one word", func(t *testing.T) {
-			t.Parallel()
-			give := "* (no branch, rebasing feature)"
-			have := git.ParseActiveBranchDuringRebase(give)
-			want := gitdomain.NewLocalBranchName("feature")
-			must.Eq(t, want, have)
-		})
-		t.Run("branch name is two words", func(t *testing.T) {
-			t.Parallel()
-			give := "* (no branch, rebasing feature branch)"
-			have := git.ParseActiveBranchDuringRebase(give)
-			want := gitdomain.NewLocalBranchName("feature branch")
-			must.Eq(t, want, have)
-		})
-		t.Run("branch name is three words", func(t *testing.T) {
-			t.Parallel()
-			give := "* (no branch, rebasing the feature branch)"
-			have := git.ParseActiveBranchDuringRebase(give)
-			want := gitdomain.NewLocalBranchName("the feature branch")
-			must.Eq(t, want, have)
-		})
-	})
```

Comment:
I just realized there's a cleaner way to test this, and would also validate your new implementation. I just shipped [#4760](https://github.com/git-town/git-town/pull/4760) with these improvements.

I'm holding off on syncing your branch myself to avoid interfering with your changes. When you do pull in the latest main, you might hit some merge conflicts. You should be able to keep all your production code changes and safely discard mine during the merge.

Just a heads up: the `currentBranchDuringRebase` method is now capitalized to make it testable, so it moved to a different spot in the file.